apps/src/gamelab/TextConsole.jsx

```diff
@@ -1,10 +1,58 @@
 /** @file Grid over visualization */
 import PropTypes from 'prop-types';
 import React from 'react';
+import {Transition} from 'react-transition-group';
 
 export const styles = {
   hide: {
     display: 'none'
+  },
+  console: {
+    display: '',
+    background: 'rgba(128,128,128,0.3)',
```

Comment:
(here and elsewhere) How were these constants decided?

Reply:
Here: they match the prototype made by JoshC (actually, they were slightly off, fixed now). You can see the prototype in the old version of  dashboard/config/libraries/SpriteLab2Beta.interpreted.js 
Other places: I added some consts that should make that clearer.